src/schema/v2/artworksForUser/artworksForUser.ts

```diff
@@ -80,7 +83,8 @@ export const artworksForUser: GraphQLFieldConfig<void, ResolverContext> = {
       "id"
     ).slice(0, size)
 
-    const totalCount = filteredArtworkIds.length + backfillArtworks.length
+    const totalCount =
+      filteredArtworkIds.length + (backfillArtworksTotalCount ?? 0)
```

Comment:
This is the actual fix. Instead of using the length of the sliced `backfillArtworks` we need to add `backfillArtworksTotalCount`